components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -3445,9 +3494,9 @@ public APIResponseFaultCount(OMElement row) {
                     APIUsageStatisticsClientConstants.INVOCATION_TIME));
             OMElement faultCountEle = row.getFirstChildWithName(new QName(
                     APIUsageStatisticsClientConstants.FAULT));
-            if (invocationTimeEle != null) {
-                requestTime = invocationTimeEle.getText();
-            }
+//            if (invocationTimeEle != null) {
```

Comment:
already removed the code in commit https://github.com/wso2/carbon-apimgt/commit/078e20af796c5e61f8f9696b995e05cb82ff8953